mxnet-spring-samples-image-recognition/pom.xml

```diff
@@ -55,8 +55,8 @@
         -->
         <dependency>
             <groupId>org.apache.mxnet</groupId>
-            <artifactId>mxnet-full_2.11-linux-x86_64-cpu</artifactId>
-            <version>${mxnet-cpu.version}</version>
+            <artifactId>mxnet-full_2.11-osx-x86_64-cpu</artifactId>
```

Comment:
Here's a way to make this pom file system agnostic for linux and OSX. 

https://github.com/awslabs/deeplearning-benchmark/blob/master/scala-mxnet/scala-bm/pom.xml#L21

Reply:
@tomoncle, yea I wouldn't merge... @piyushghai is write, we need a way to have it os diagnostic. 

@piyushghai, looked into it, ah yes...that's much nicer. I tried with the link you sent in the last comment and works nicely when I tested it. 

When I tried to add the `<profile>` section from the link sent under dependencies in `mxnet-spring-samples-image-recognition/pom.xml`, and add the `<properties>` info as well, I am not able to import the mac osx dependencies sadly. Since I removed my repo with these changes by mistake and don't see a way to pull down the changes, I just copy pasta'd what I have here: https://pastebin.com/raw/KEywRekE ... maybe it'd be easier to create a new PR. 

Might be easier to create a new PR.

Reply:
It seems to be the best way to do it.